docs/config.md

```diff
@@ -153,6 +153,12 @@ Important environment variables include:
 |-|-|-|
 | `OTEL_EXPORTER_ZIPKIN_ENDPOINT` | Zipkin URL. | `http://localhost:8126` |
 
+## Instrumentation options
+
+| Environment variable | Description | Default value |
+|-|-|-|
+| `OTEL_DOTNET_AUTO_GRAPHQL_SET_DOCUMENT` | Whether GraphQL instrumentation can pass raw queries as `graphql.document` attribute. This may contain sensitive information and therefore is disabled by default. | `false` |
```

Comment:
It would not hurt to mention it in changlog especially that the default behavior is different

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1265/commits/55c0c0084f9d4540ce97f9ca3217071a155a9044